aws-codeguruprofiler-profilinggroup/sample-template.json

```diff
@@ -5,27 +5,49 @@
     "MyProfilingGroup": {
       "Type": "AWS::CodeGuruProfiler::ProfilingGroup",
       "Properties": {
-        "ProfilingGroupName": "MySampleProfilingGroup"
+        "ProfilingGroupName": "MySampleProfilingGroup10",
+        "ComputePlatform": "Default",
+        "AnomalyDetectionNotificationConfiguration": [
+          {
+            "channelUri": "arn:aws:sns:us-east-1:977364881114:MyTopic"
+          }
+        ]
       }
     },
     "MyProfilingGroupWithDefaultComputePlatform": {
       "Type": "AWS::CodeGuruProfiler::ProfilingGroup",
       "Properties": {
-        "ProfilingGroupName": "MyProfilingGroupWithDefaultComputePlatform",
-        "ComputePlatform": "Default"
+        "ProfilingGroupName": "MyProfilingGroupWithDefaultComputePlatform10",
+        "ComputePlatform": "Default",
+        "AnomalyDetectionNotificationConfiguration": [
+          {
+            "channelUri": "arn:aws:sns:us-east-1:977364881114:MyTopic"
+          }
+        ]
       }
     },
     "MyProfilingGroupWithLambdaComputePlatform": {
       "Type": "AWS::CodeGuruProfiler::ProfilingGroup",
       "Properties": {
-        "ProfilingGroupName": "MyProfilingGroupWithLambdaComputePlatform",
-        "ComputePlatform": "AWSLambda"
+        "ProfilingGroupName": "MyProfilingGroupWithLambdaComputePlatform10",
+        "ComputePlatform": "AWSLambda",
+        "AnomalyDetectionNotificationConfiguration": [
+          {
+            "channelUri": "arn:aws:sns:us-east-1:977364881114:MyTopic"
+          }
+        ]
       }
     },
     "MyProfilingGroupWithAgentPermissions": {
       "Type": "AWS::CodeGuruProfiler::ProfilingGroup",
       "Properties": {
-        "ProfilingGroupName": "MySampleProfilingGroupWithAgentPermissions",
+        "ProfilingGroupName": "MySampleProfilingGroupWithAgentPermissions10",
+        "ComputePlatform": "Default",
+        "AnomalyDetectionNotificationConfiguration": [
+          {
+            "channelUri": "arn:aws:sns:us-east-1:977364881114:MyTopic"
+          }
+        ],
         "AgentPermissions": {
```

Comment:
Sorry about this, I forgot to upload a revision and prematurely marked the comments as resolved to help comb through them for addressing. The one part about the sample template is that if we add a fake id, when a user tries to use this template, it will fail since that URI will not be valid. Are we ok with this? If so, what you suggested seems like a good idea

Reply:
Wouldn't the current one fail as well, given that `977364881114` is your personal account? E.g. it would fail as well with `arn:aws:sns:us-east-1:123456789012:MyTopic` as it does with that one, unless I'm missing your question?